tools/depends/target/pythonmodule-pil/Makefile

```diff
@@ -41,23 +46,26 @@ $(TARBALLS_LOCATION)/$(ARCHIVE):
 $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	rm -rf $(PLATFORM)/*; mkdir -p $(PLATFORM)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
+ifeq ($(TARGET_PLATFORM),raspberry-pi)
+	cd $(PLATFORM); patch -p1 -i ../01-rbpi-remove-lcms.patch
```

Comment:
Why patch the build system instead of using the already existing configure option `--disable-lcms`?

Reply:
Because it doesnt work on rbpi platform. No idea why. Works fine on every other one.